### DISCUSSION TITLE
Key Value store in place of Redis

### DISCUSSION THREAD

**Discussion by vibhuh** [rosedblabs/rosedb#320] [Q&A] [answered]
I am developing an Accounting and Inventory Cloud application with Golang as the backend. Currently using Redis for storing Access and Refresh Token Keys with UserID as values. I heard about RoseDB now, I just wanted to know if there is any performance advantage over Redis. Our requirement is very limited to save users jwt tokens with userID as values.

It will be much helpful, if you can share your insights.

Thanks and Regards,

**Comment by roseduan** [comment, marked as answer]
Hi.

RoseDB is based on a very simple storage model, put a new key/value pair consists of a disk write(append only) and a memory update.
And this is almost the same with redis(depends on Redis AOF strategy).
So the performance maybe almost the same, you can write a simple benchmark.
